app/views/staff/session_types/update.js.erb

```diff
@@ -0,0 +1,20 @@
+$('#session-type-edit-<%= session_type.id %>').modal('hide');
+
+var cells = $('table#session-types #session_type_<%= session_type.id %> td');
```

Comment:
All the rest of this could be replaced by something like this to reuse existing partials:

```
var tableRow = $('table#session-types #session_type_<%= session_type.id %>');
tableRow.replaceWith('<%=j render(session_type) %>');
```